Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Person.h"
+
+#include <iostream>
+
+Person::Person()
+{
+  this -> name_ = "";
+  this -> age_ = new int(0);
+}
+
+Person::Person (const std::string& name, int age)
+{
+  this -> name_ = name;
+  this -> age_ = new int(age);
```

Comment:
We generally prefer using [Constructor Initializer Lists](https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists)

Reply:
Hey Liam! Made the changes as you've requested, please feel free to take a look when you can thanks.